website/src/_posts/2021-12-2.1-2.3.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Uppy 2.1–2.3 or Recently in the Uppy Universe: Audio plugin, Unsplash update, switch to Yarn, Status Bar improvements and more" 
+date: 2021-12-15
+author: 
+  - aduh95
+  - murderlon
+  - arturi
+  - mifi
+image: https://uppy.io/images/blog/2.1-2.3/audio-cover.jpg
+published: true
+---
+
+After the 2.0 release we’ve been busy refactoring, updating dependencies, moving from npm to Yarn 3 and attending to much-needed housekeeping. We’ve released a new Audio plugin, improved Status Bar and Unsplash plugins and made a few significant improvements to Companion.
+
+<!--more-->
+
+## Audio plugin
+
+Uppy now has a new `@uppy/audio` plugin for recording audio memos, messages and sounds. Check it out [on the demo page](https://uppy.io/examples/dashboard/).
```

Comment:
By saying "messages and sounds" it kind of implies you can add messages in some other form that audio?